ra/ra.go

```diff
@@ -1152,12 +1160,32 @@ func (ra *RegistrationAuthorityImpl) validateFinalizeRequest(
 			ID:            authz.ID,
 			ChallengeType: solvedByChallengeType,
 		}
+
+		// Count the number of authorizations where the original CAA check was
+		// reused and the number where it was rechecked.
+		staleCAA, err := validatedBefore(authz, ra.clk.Now().Add(caaRecheckDuration))
+		if err != nil {
+			// This should never happen because we just checked the same thing
+			// inside ra.checkOrderAuthorizations.
```

Comment:
Why not move this logging code into `ra.checkOrderAuthorizations`? There we can take advantage of the existing calculation. We would need to add `registrationID` to its parameters, but that's probably good anyhow.

(We could deduce registrationID from the field in all the authzs that are passed, but better explicit over implicit).

Reply:
Very interesting! While this approach doesn't necessarily change the number of times we loop over the authzs I do appreciate that it moves this operation closer to where that logic could change and gives us a convenient way to obtain a count.